README.md

```diff
@@ -57,7 +57,6 @@ Server code is located at [ottomated/CrewLink-server](https://github.com/ottomat
 
 ### Prerequisites
 
-This is an example of how to list things you need to use the software and how to install them.
 * [Python](https://www.python.org/downloads/)
```

Comment:
Is python even an actual prereq for this repo? Granted I do have python installed, but I'm not sure it's actually needed...

Reply:
Looks like it's required for memoryjs/node-gyp, per https://github.com/ottomated/CrewLink/pull/72